test/specs/comment/tests/01.comment.js

```diff
@@ -0,0 +1,30 @@
+exports.execute = function (options) {
+
+    describe('For checking the .comment on models, ', function () {
+        var schemaName = "schema_with_comment", schema;
+
+        beforeAll(function (done) {
+            schema = options.catalog.schemas.get(schemaName);
+            done();
+        });
+
+        // Test Cases:
+        it('schema_with_comment should have a non-null .comment property.', function () {
+            expect(schema.comment).toBe("schema with a comment");
```

Comment:
You might want to add asserts for checking whether **schema** has **comment** variable set in its class, i.e. `schema.hasOwnProperty("comment")`. This was we could make sure that the property exists on particular classes.

The same applies for **table**, **foreignKey** and **key**.